webhook_router/app.py

```diff
@@ -73,11 +38,12 @@ def handle_github_webhook() -> tuple[str, int]:
             return "Signature validation failed!", 403
     payload = request.get_json()
     app.logger.debug("Received webhook: %s", payload)
-    webhook_logger.log(logging.INFO, json.dumps(payload))
+    app.logger.info(json.dumps(payload))
     return "", 200
 
 
-if __name__ == "__main__":
+# Exclude from coverage since unit tests should not run as __main__
+if __name__ == "__main__":  # pragma: no cover
     # Start development server
     app.logger.addHandler(logging.StreamHandler(sys.stdout))
     app.logger.setLevel(logging.DEBUG)
```

Comment:
I removed the streamhandler but kept the log level setting as I am reluctant to enable debug mode due to https://bandit.readthedocs.io/en/1.7.8/plugins/b201_flask_debug_true.html. I know that its config for a dev server but it could inadvertently be used for production.

Reply:
Understand, thanks!